pom.xml

```diff
@@ -4,17 +4,19 @@
 	<artifactId>eam-light-backend</artifactId>
 	<version>3.5.27-SNAPSHOT</version>
 	<packaging>pom</packaging>
+
 	<modules>
 		<module>eam-light-backendear</module>
 		<module>eam-light-backendweb</module>
 		<module>eam-light-backendejb</module>
+		<module>eam-light-cern</module>
+		<module>eam-light-opensource</module>
+		<module>eam-light-shared</module>
 	</modules>
 
 	<properties>
 		<artifact.filename>${project.build.finalName}-${project.version}</artifact.filename>
-        <ch.cern.cmms.eamtools.version>2.3.2</ch.cern.cmms.eamtools.version>
-		<ch.cern.cmms.ldaptools.version>0.1.4</ch.cern.cmms.ldaptools.version>
-		<ch.cern.eam.eam-wshub-core.version>11.5-28</ch.cern.eam.eam-wshub-core.version>
+		<ch.cern.eam.eam-wshub-core.version>11.5-29-SNAPSHOT</ch.cern.eam.eam-wshub-core.version>
```

Comment:
Remember to change core version to non-snapshot one before merge.

Reply:
Updated to 11.6-2